.github/actions/create-bwc-build/action.yaml

```diff
@@ -0,0 +1,47 @@
+name: 'Create a backwards compatible ready build'
+description: 'Checkouts the official version of a the Security plugin and builds it so it can be used for BWC tests'
+
+inputs:
+  plugin-branch:
+    description: 'The branch of the plugin that should be built, e.g "2.2", "1.x"'
+    required: true
```

Comment:
I have not needed to specify the security version in the past since we care on a given version branch. Is this meant to be the backwards version we are testing? Maybe rename if so.

Reply:
This allows the tests to be run against arbitrary builds, rather than only versions that are build on build.ci.opensearch.org.  We could run BWC test on windows if 2.3 builds on windows even if we don't have an official build.

Reply:
Oh okay. This makes sense then. Thank you for explaining.